### PR TITLE
Update build and project structure

### PR DESCRIPTION
**Pull request by cmmclaug**
Lots of stuff here, mostly just integration stuff.
* The summary_report is a dummy showing how to load outputs for now
* The NBconvert output isn't pretty...will have to address

### CODE REVIEW

File: src/generate_eda.py [hunk 1]
```diff
@@ -69,8 +83,7 @@ def generate_dataset_overview(data_frame, output_folder, file_name):
         
     Returns:
     -----------
-    str
-        saved file path
+    None
```
Comment (zmerpez) [inline src/generate_eda.py:90]: that does return data frame now. If we need to use it somewhere as a function

File: src/generate_eda.py [hunk 2]
```diff
@@ -100,8 +117,7 @@ def generate_feature_overview(data_frame, output_folder, file_name):
         
     Returns:
     -----------
-    object
-        saved .png file
+    None
```
Comment (zmerpez) [inline src/generate_eda.py:124]: same returns data frame for features overview

**Review by zmerpez — APPROVED** [review]
Code seems legit. I had 2 small comments for generate_eda.py. I still need to run and check if everything goes as planned. Thanks for combining all in a single command file, this seems to be necessary for milestone 3.

**Review by aidanmattrick — COMMENTED** [review]
This looks good to me Craig! Thanks for writing this so we don't have to continually compile source code by hand.

**Review by ssyayayy — APPROVED** [review]
All appears good

File: src/generate_eda.py (hunk 1 above)
Comment (cmmclaug) [reply src/generate_eda.py:90]: I modified to export the dataframe only, so we don't actually return anything!

File: src/generate_eda.py (hunk 2 above)
Comment (cmmclaug) [reply src/generate_eda.py:124]: Not anymore :)

**Pull request merged**